### ISSUE TITLE
Changes filename of DB to lowercase

### ISSUE THREAD

**Issue by jvlatzko** [PhilippC/keepass2android#1121]
Hi, 

I've recently saved the database using keepass2android. (I needed to create a new entry to DB on the way) Application saved file in lowercase. Desktop application broke over this (obviously easy to fix, but still). 

Prior: XY.kdbx
After: xy.kdbx

Haven't found the place where keepass2android saves, otherwise I'd have submitted a PR.. 

Best
Vincent

**Comment by gabrielfin** [comment]
I'm having the same issue. Everytime I save my database, which contains uppercase letters, it is converted to lowercase.
It didn't use to happen before, so it must have broken in the last version.

**Comment by codeThatThinks** [comment]
This is a duplicate of #898
